### DISCUSSION TITLE
Can we use ZLUDA to run a Python script such as in Gaussian Splatting?

### DISCUSSION THREAD

**Discussion by febrifahmi** [vosen/ZLUDA#275] [General]
Hi,

I would like to know the possibility of using ZLUDA to run a special Python script just like the one in 3D Gaussian Splatting repository such as: `zluda.exe -- python train.py <application_arguments>` or `zluda_with.exe -- python train.py <application_arguments>`; is it possible or not? 

Since it is not running the exe directly but running the script via Python.exe on the computer that has no CUDA capable GPU and thus won't have that specific CUDA GPU driver installed, I'm afraid that it will triggered "no CUDA GPU found" error during execution since it heavily depends on torch and many other libraries which may be or may be not run it ZLUDA way.

I hope you could give me a hint.

Thanks!

**Comment by vosen** [comment]
There should be no issue with running python, it's an executable like any other. On the other hand, if it uses pytoch it probably will not work too well. PyTorch is massive and ZLUDA supports only a tiny subset of it.
